boards/BMS/Src/App/App_TractiveSystem.c

```diff
@@ -51,3 +51,18 @@ float App_TractiveSystem_GetLowResCurrent(struct TractiveSystem *ts)
 {
     return ts->get_low_res_current(ts->get_adc_low_res_current());
 }
+
+float App_TractiveSystem_GetCurrent(struct TractiveSystem *ts)
+{
+    float low_res_current  = App_TractiveSystem_GetLowResCurrent(ts);
+    float high_res_current = App_TractiveSystem_GetLowResCurrent(ts);
```

Comment:
Both functions are GetLowResCurrent at the moment

Reply:
Do you think the user will need access to either of these functions? In my opinion App_TractiveSystem_GetCurrent will suffice

Reply:
If that is the case we should remove these functions and implement them directly in `App_TractiveSystem_GetCurrent`

Reply:
true

Reply:
I think having them available is good to tune the hardware or something. It would be good to check both. But I will only broadcast the one message.

Reply:
alright lets change to this for now

```suggestion
    float low_res_current  = App_TractiveSystem_GetLowResCurrent(ts);
    float high_res_current = App_TractiveSystem_GetHighResCurrent(ts);
```